### PR TITLE
add support for multiple characters

### PR DESCRIPTION
**Pull request by reimertz**
## Implementation
Just added the support for >1 character in the mention plugin

## Demo
mention

## Use-case

I needed this plugin to support more than one characters so I implemented it.. 😄

### CODE REVIEW

File: draft-js-mention-plugin/src/MentionSuggestions/index.js [hunk 1]
```diff
@@ -131,8 +131,8 @@ export default class MentionSuggestions extends Component {
     const selectionIsInsideWord = leaves
       .filter((leave) => leave !== undefined)
       .map(({ start, end }) => (
-        (start === 0 && anchorOffset === 1 && plainText.charAt(anchorOffset) !== this.props.mentionTrigger && new RegExp(this.props.mentionTrigger, 'g').test(plainText) && anchorOffset <= end) || // @ is the first character
-        (anchorOffset > start + 1 && anchorOffset <= end) // @ is in the text or at the end
+        (start === 0 && anchorOffset === this.props.mentionTrigger.length && plainText.charAt(anchorOffset) !== this.props.mentionTrigger && new RegExp(this.props.mentionTrigger, 'g').test(plainText) && anchorOffset <= end) || // @ is the first character
```
Comment (juliankrispel) [inline draft-js-mention-plugin/src/MentionSuggestions/index.js]: this line is getting a bit long, are you not getting an eslint error here?

**Comment by reimertz** [comment]
Ty! Ran lint, no issues! :)

**Comment by juliankrispel** [comment]
just tested, lgtm

image

@reimertz can you update the changelog?

**Comment by reimertz** [comment]
@juliankrispel I really don't care but any reason why you don't merge the actual PR's rather than merging/committing them yourself? 

image

**Comment by juliankrispel** [comment]
@reimertz I did merge the actual pr:

image

Anything I an help with?

**Comment by reimertz** [comment]
@juliankrispel I wonder what's different in your flow then since when I merge PR's their commit/contributions are added to their account rather than attached to my own. 🤔

Example:
https://github.com/draft-js-plugins/draft-js-plugins/commits?author=juliankrispel

**Comment by juliankrispel** [comment]
Your commits are still yours. Github simply creates an extra commit for the merge itself

Julian Krispel
UI Engineer & Consultant
http://reactrocket.com
+447988116178

**Pull request merged**